lib/elixir/pages/mix-and-otp/agents.md

```diff
@@ -47,7 +42,7 @@ iex> Agent.stop(agent)
 :ok
 ```
 
-We started an agent with an initial state of an empty list. We updated the agent's state, adding our new item to the head of the list. The second argument of `Agent.update/3` is a function that takes the agent's current state as input and returns its desired new state. Finally, we retrieved the whole list. The second argument of `Agent.get/3` is a function that takes the state as input and returns the value that `Agent.get/3` itself will return. Once we are done with the agent, we can call `Agent.stop/3` to terminate the agent process.
+We started an agent with an initial state of an empty list. The `start_link/1` function returned the `:ok` tuple with a process identifier (PID) of the agent. We will use this PID for all further interactions. We then updated the agent's state, adding our new item to the head of the list. The second argument of `Agent.update/3` is a function that takes the agent's current state as input and returns its desired new state. Finally, we retrieved the whole list. The second argument of `Agent.get/3` is a function that takes the state as input and returns the value that `Agent.get/3` itself will return. Once we are done with the agent, we can call `Agent.stop/3` to terminate the agent process.
```

Comment:
I see that we now explain the `child_spec` concept in a later guide, which is great. I can't think of the right wording currently, but I think we should hint at it a small bit here to make it clear that `start_link/1` is more than just a convention, but something that Elixir expects to be defined so that it knows how to start this agent. 

Tried to use generic wording there to avoid explaining supervisors and child specs, but the important bit is that `start_link/1` should exist and should be called `start_link/1` and its more than just a naming convention.